oqamaint/openqabot.py

```diff
@@ -496,16 +496,18 @@ def incident_openqa_jobs(self, s):
                 'latest': '1'
             })['jobs']
 
-    # for SUSE we use mesh for openSUSE we limit the jobs to open release requests
+    # for SUSE we use mesh, for openSUSE we limit the jobs to open release requests
     def check_opensuse_incidents(self):
-        requests = []
+        requests = dict() # collecting unique requests
         self.wait_for_build = set()
         for prj in self.tgt_repo[self.openqa.baseurl].keys():
-            requests += self.ids_project(prj, 'maintenance_release')
+            for r in self.ids_project(prj, 'maintenance_release'):
+                requests[r.reqid] = r
 
         # to be stored in settings
         issues = dict()
-        for req in requests:
+        for req in sorted(requests.keys()):
```

Comment:
why sorted ?

Reply:
to make it easier for me to follow the debug output - no other reason